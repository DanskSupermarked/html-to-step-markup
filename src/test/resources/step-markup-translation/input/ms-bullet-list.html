<UL style="MARGIN-TOP: 0cm" type=disc>
<LI style="MARGIN: 0cm 0cm 0pt; mso-list: l0 level1 lfo1; tab-stops: list 36.0pt" class=MsoNormal><FONT size=3><FONT face="Times New Roman">Skal af Termoplast<?xml:namespace prefix = o ns = "urn:schemas-microsoft-com:office:office" /><o:p></o:p></FONT></FONT></LI>
<LI style="MARGIN: 0cm 0cm 0pt; mso-list: l0 level1 lfo1; tab-stops: list 36.0pt" class=MsoNormal><FONT size=3><FONT face="Times New Roman">Indbygget solbrille<o:p></o:p></FONT></FONT></LI>
<LI style="MARGIN: 0cm 0cm 0pt; mso-list: l0 level1 lfo1; tab-stops: list 36.0pt" class=MsoNormal><FONT size=3><FONT face="Times New Roman">Komfortabel hjelm til de lange ture<o:p></o:p></FONT></FONT></LI>
<LI style="MARGIN: 0cm 0cm 0pt; mso-list: l0 level1 lfo1; tab-stops: list 36.0pt" class=MsoNormal><FONT size=3><FONT face="Times New Roman">Aftageligt og vaskbart SANITIZED interiør<o:p></o:p></FONT></FONT></LI>
<LI style="MARGIN: 0cm 0cm 0pt; mso-list: l0 level1 lfo1; tab-stops: list 36.0pt" class=MsoNormal><FONT size=3><FONT face="Times New Roman">Quick-release hageremslås<o:p></o:p></FONT></FONT></LI>
<LI style="MARGIN: 0cm 0cm 0pt; mso-list: l0 level1 lfo1; tab-stops: list 36.0pt" class=MsoNormal><FONT size=3><FONT face="Times New Roman">Smart kæbeåbningssystem<o:p></o:p></FONT></FONT></LI>
<LI style="MARGIN: 0cm 0cm 0pt; mso-list: l0 level1 lfo1; tab-stops: list 36.0pt" class=MsoNormal><FONT size=3><FONT face="Times New Roman">Ridsefast, anti-dug visir<o:p></o:p></FONT></FONT></LI>
<LI style="MARGIN: 0cm 0cm 0pt; mso-list: l0 level1 lfo1; tab-stops: list 36.0pt" class=MsoNormal><FONT size=3><FONT face="Times New Roman">Ventilationssystem<o:p></o:p></FONT></FONT></LI></UL>
