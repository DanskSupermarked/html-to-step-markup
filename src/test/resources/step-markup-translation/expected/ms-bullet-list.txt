<bulletlist>
<bullet>Skal af Termoplast</bullet>
<bullet>Indbygget solbrille</bullet>
<bullet>Komfortabel hjelm til de lange ture</bullet>
<bullet>Aftageligt og vaskbart SANITIZED interiør</bullet>
<bullet>Quick-release hageremslås</bullet>
<bullet>Smart kæbeåbningssystem</bullet>
<bullet>Ridsefast, anti-dug visir</bullet>
<bullet>Ventilationssystem</bullet>
</bulletlist>
